src/test/groovy/com/jvm_bloggers/core/rss/converters/SyndFeedJsonMessageConverterSpec.groovy

```diff
@@ -57,6 +61,6 @@ class SyndFeedJsonMessageConverterSpec extends Specification {
         def actual = message.getBodyAsString(Charset.defaultCharset())
         def expected = IOUtils.toString(getClass().getResource("expected-rss.json").openStream(), Charset.defaultCharset())
 
-        actual == expected
+        toJson(actual) == toJson(expected)
```

Comment:
This change is required in order to compare produced jsons independently of key order.